<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortSubDirsDialog</class>
 <widget class="QDialog" name="SortSubDirsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sort the Sub Directorys</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="SortSubDirsWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>36</y>
     <width>77</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="sortSubDirsBtn">
    <item>
     <widget class="QPushButton" name="moveUpBtn">
      <property name="text">
       <string>Move Up</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="moveDownBtn">
      <property name="text">
       <string>Move Down</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>30</y>
     <width>321</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="sortSubDirsTree">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Parent folder:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dirlabel">
      <property name="text">
       <string>dir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
